core/src/main/java/com/microsoft/applicationinsights/internal/schemav2/Envelope.java

```diff
@@ -139,8 +139,11 @@ public double getSampleRate() {
     /**
      * Sets the SampleRate property.
      */
-    public void setSampleRate(double value) {
-        this.sampleRate = value;
+    public void setSampleRate(Double value) {
```

Comment:
this file is autogenerated. Instead of changing this file - can you please fix the code that attempts to set this to `null` or resulting code that fails because of unexpected value.

Alternatively you'd need to modify code generator, but I think it's better to make this code works with `null` value